src/Tests/Riganti.Selenium.Core.Samples.Common.Tests/AppSeleniumTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Riganti.Selenium.AssertApi;
+using Riganti.Selenium.Core.Abstractions;
+using Xunit.Abstractions;
+
+namespace Riganti.Selenium.Core.Samples.AssertApi.Tests
+{
+    public abstract class AppSeleniumTest : SeleniumTest
+    {
+        public AppSeleniumTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        protected void RunInAllBrowsers(Action<IBrowserWrapper> testBody, [CallerMemberName]string callerMemberName = "",
+            [CallerFilePath]string callerFilePath = "", [CallerLineNumber]int callerLineNumber = 0)
+        {
+            AssertApiSeleniumTestExecutorExtensions.RunInAllBrowsers(this, testBody, callerMemberName, callerFilePath, callerLineNumber);
+        }
+    }
+}
```

Comment:
Why would you use a base class when you are just sharing code, not state. I think that calling a function is much more appropriate.

<!-- Sorry for flaming here, but this is a utter bullshit. You are using inheritance instead of using static - just to save few characters for using one special feature. If you'd need to use two features like this, would you use multiple inheritance?
![hovno](https://user-images.githubusercontent.com/7894687/42992086-1811f30c-8bf7-11e8-8f48-ccc8753f1d94.jpg) -->